src/entities/therapy-session.entity.ts

```diff
@@ -0,0 +1,46 @@
+import { Column, Entity, JoinTable, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { SIMPLYBOOK_ACTION_ENUM } from '../utils/constants';
+import { BaseEntity } from './base.entity';
+import { PartnerAccessEntity } from './partner-access.entity';
+
+@Entity({ name: 'therapy_session' })
+export class TherapySessionEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  action: SIMPLYBOOK_ACTION_ENUM;
+
+  @Column({ unique: true })
+  client_email: string;
```

Comment:
the `unique` should be removed here and in the migration otherwise it'll have `unique` on the db